doc/website/blog/2022-06-announcing-cockroachdb-support-preview.md

```diff
@@ -0,0 +1,120 @@
+[Terraform integration](https://atlasgo.io/blog/2022/05/04/announcing-terraform-provider). By using Atlas's
+Data Definition Language (with a syntax similar to Terraform), users can plan, verify and apply changes
+to their databases in a simple, declarative workflow.
+Earlier this year, Atlas became the [migration engine for Ent](https://entgo.io/blog/2022/01/20/announcing-new-migration-engine),
+a widely popular, Linux Foundation backed entity framework for Go.
+
+## CockroachDB
+[CockroachDB](https://www.cockroachlabs.com/) is an [open-source](https://github.com/cockroachdb/cockroach) NewSQL 
+database. From their README:
+> CockroachDB is a distributed SQL database built on a transactional and strongly-consistent 
+> key-value store. It scales horizontally; survives disk, machine, rack, and even datacenter 
+> failures with minimal latency disruption and no manual intervention; supports strongly-consistent 
+> ACID transactions; and provides a familiar SQL API for structuring, manipulating, and querying data.  
+  
+CockroachDB has been gaining popularity and many of you [have](https://github.com/ent/ent/issues/2545)
+[been](https://github.com/ariga/atlas/issues/785#issue-1231951038) [asking](https://github.com/ariga/atlas/issues/785#issuecomment-1125853135) 
+for Atlas to support it.
+
+While CockroachDB aims to be PostgreSQL compatible, it still has some incompatibilities 
+(e.g. [1](https://github.com/cockroachdb/cockroach/issues/20296#issuecomment-1066140651), 
+[2](https://github.com/cockroachdb/cockroach/issues/82064),[3](https://github.com/cockroachdb/cockroach/issues/81659))
+which prevented Atlas users from using the existing Postgres dialect from working with it.  
+  
+With the latest release of Atlas, the Postgres driver automatically detects when it is connected to a CockroachDB
+database and uses a custom driver which provides compatability with CockroachDB.
+
+### Getting started with Atlas and CockroachDB
+
+Let's see how we can use Atlas CLI to manage the schema of a CockroachDB database. 
+Start by downloading the latest version of Atlas, on macOS:
+```
+brew install ariga/tap/atlas
+```
+For installation instructions on other platforms, see [the docs](https://atlasgo.io/cli/getting-started/setting-up#install-the-cli).
+
+For the purpose of this example, let's spin up a local, [single-node CockroachDB cluster](https://github.com/cockroachlabs-field/cockroachdb-single-node)
+in a container by running:
+```
+docker run -it -p 8080:8080 -p 26257:26257 -e "DATABASE_NAME=test" -e "MEMORY_SIZE=.5" timveil/cockroachdb-single-node:latest
+```
+
+Next, use Atlas's `schema inspect` command to read the schema of our local database and save the result to a file:
+```
+atlas schema inspect -u 'postgres://root:pass@localhost:26257/?sslmode=disable' --schema public > schema.hcl
+```
+Observe the current HCL representation of the `public` schema, empty with no tables:
+```hcl
+schema "public" {
+}
+```
+
```

Comment:
This seems like an unnecessary step. Once can just create the file manual in the step below, where it gets edited anyway. Inspect afterwards is cooler I think. It demonstrates more of what the tool can do.